tests/integration/functional/model/test_model_version.py

```diff
@@ -217,3 +225,67 @@ def test_tags_properly_updated(self):
                 model = mv._get_or_create_model()
                 assert len(model.tags) == 2
                 assert {t.name for t in model.tags} == {"foo", "bar"}
+
+    def test_metadata_logging(self):
```

Comment:
Could you add testing for the failure modes of this logging?

Reply:
I have it in `test_metadata_logging_functional`, it cannot fail in the object-oriented way on metadata logging (unless unexpected issues), only failing before the logging is possible, e.g. cannot get ModelVersion or else.